[Format]Fixed an Issues with adding dot at the end of description (#3630)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## Unreleased

> * Fixed an issue where **format** add unnecessary period at the end of descriptions that ending with brackets that contains a dot at the end of them.
> * Fixed an issue where **format** was not adding a period at the end of description filed in script yml files.

Reviewer: ```suggestion
* Fixed an issue where **format** would add unnecessary period at the end of descriptions ending with brackets.
* Fixed an issue where **format** would not add a period at the end of descriptions, when running on in script files.
```
